src/components/banner/banner.spec.ts

```diff
@@ -0,0 +1,250 @@
+import {
+  elementUpdated,
+  expect,
+  fixture,
+  html,
+  unsafeStatic,
```

Comment:
Nit: `unsafeStatic` is rarely needed when testing a concrete component. You can template with the default `html` function.